File: python/class5/guess_my_number/guess_my_number.py
import random

print "\n\t<<<< GUESS MY NUMBER : THE GAME >>>>\n"
        
random_value = random.randint(1,20) # [1,20] # no need to add 1 like in the java example
        
user_input = input("\nPlease guess my number (1-20): ")

if user_input == random_value:
    print "\n\tYOU DID IT!!!\n"
else:
    print "\n\tYOU'RE WRONG!!!   >:(\n"
    print "\tThe number was: ", random_value, "\n"
    
    new_random_value = random.randint(1,20)
    while random_value == new_random_value:
        new_random_value = random.randint(1,20)
        
    print "\tThe number was also definitely not: ", new_random_value, "\n"

File: java/class5/GuessMyNumber/GuessMyNumber.java
import java.util.Random;
import java.util.Scanner;

public class GuessMyNumber {

    public static void main(String[] args) {
        
        System.out.println("\n\t<<<< GUESS MY NUMBER : THE GAME >>>>\n");
        
        Random rand = new Random();
        int random_value = rand.nextInt(20) + 1;
        
        System.out.print("\nPlease guess my number (1-20): ");
        Scanner keyboard = new Scanner(System.in);
        int user_input = keyboard.nextInt();
        
        if (user_input == random_value) {
            System.out.println("\n\tYOU DID IT!!!\n");
        }
        else {
            System.out.println("\n\tYOU'RE WRONG!!!   >:(\n");
            System.out.println("\tThe number was: " + random_value + "\n");
            
            int new_random_value = rand.nextInt(20) + 1;
            while (random_value == new_random_value) {
                new_random_value = rand.nextInt(20) + 1;
            }
            System.out.println("\tThe number was also definitely not: " + new_random_value + "\n");
        }
    }
    
    
}
